src/app/components/edit-resource/edit-resource.component.html

```diff
@@ -174,13 +174,9 @@
     </div>
   </div>
 
-  <div class="row">
+  <div class="row" *ngIf="!isMetaTool()">
     <div class="col">
       <label>Manifest XML</label>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col">
```

Comment:
Not sure I understand this change.

Reply:
oh, this was collapsing 2 rows into a single row to match the rest of the layout. It was some cleanup I probably should have done as a separate commit

Reply:
previously the label and the form field were in 2 separate rows, this combines them into the same row, and just lets them naturally stack within a column in that row.